crowbar_framework/app/controllers/barclamp_controller.rb

```diff
@@ -521,6 +521,7 @@ def valid_transition_states
       "applying", "discovered", "discovering", "hardware-installed",
       "hardware-installing", "hardware-updated", "hardware-updating",
       "installed", "installing", "ready", "readying", "recovering",
+      "crowbar_upgrade",
```

Comment:
We can clean this up in a followup PR anyway, imho (the states displayed in leds / the pie are actually a different set of states than the node states, so I would just have an upgrade state there, which would include crowbar-upgrade but possibly other upgrade states).